### PR TITLE
first time return id and field

### PR DESCRIPTION
**Pull request by alive-web**

### CODE REVIEW

File: src/simple_commander/controllers/main.py [hunk 1]
```diff
@@ -36,6 +37,9 @@ class Unit(object):
     def __init__(self, x, y, angle, bonus, speed, unit_filename, bullet_filename):
         self.image_filename = unit_filename
         self.bullet_filename = bullet_filename
+        self.time_last_calculation = datetime.datetime.now()
+        self.real_x = x
```
Comment (pzdeb) [inline src/simple_commander/controllers/main.py]: let it be x1,y1 and x2,y2

File: src/command_server/command_server.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 from time import gmtime, strftime
 
 from src.core.generic import routes
-from src.simple_commander.controllers.main import GameController
+from src.simple_commander.controllers.main import GameController, STEP_INTERVAL
```
Comment (enkidulan) [inline src/command_server/command_server.py:14]: Are you really want your package to have a 'src.*' as top-level name-space?

File: src/simple_commander/controllers/main.py [hunk 3]
```diff
@@ -29,6 +30,7 @@
 
 DEFAULT_SPEED = 5
 STEP_INTERVAL = 1  # 1 second, can be changed to 0.5
+UNIT_PROPERTIES = ['x0', 'y0', 'x1', 'y1', 'angle', 'bonus', 'speed', 'shift', 'width', 'height', 'id', 'life_count']
```
Comment (enkidulan) [inline src/simple_commander/controllers/main.py:33]: this line is too long :)

File: src/simple_commander/controllers/main.py [hunk 4]
```diff
@@ -179,30 +198,32 @@ class GameController(object):
...
     def fire(self, unit):
         logging.info('Fire!! Creating bullet!')
         self.units.append(Bullet(unit))
 
-    def get_serialized_field(self):
-        result = {'field': self.game_field,
-                  'units': [unit.__dict__ for unit in self.units]}
-        return json.dumps(result)
+    def get_serialized_units(self):
+        units = [self.units[unit].to_dict() for unit in self.units]
```
Comment (enkidulan) [inline src/simple_commander/controllers/main.py:225]: WTF? you already have unit :)

Comment (alive-web) [inline src/simple_commander/controllers/main.py:225]: i will use this method when i will have units

**Pull request merged**